### PR TITLE
Italian translation of description

### PR DESCRIPTION
**Pull request by glaav**

### CODE REVIEW

File: Description/it_it.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Italiano (It)
+
+Un modpack di minecraft creato per la performance e miglioramenti grafici.
+
+[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```
Comment (Madis0) [inline Description/it_it.md:5]:
```suggestion
[![Rassegna attuale](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```

File: Description/it_it.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Fabulously Optimized [supporta quasi tutte le features della OptiFine][2], incluse le shaders, textures connesse e zoom. [Puoi anche avere il tuo mantello gratuito!][3]
+
+### Miglioramenti Minori
+
+Fabulously Optimized [include dei piccoli miglioramenti][4] che non rovinano l'esperienza vanilla, come la cronologia della chat estesa, fullscreen in modalità finestra, fixes per i bug più comuni e strumenti per trattarli.
+
+### Funziona Ovunque!
+
+Fabulously Optimized non ha mod di content, quindi funziona su tutti i server. Il modpack [può essere usato su 5 launcher, ed è hostato su 3 siti][6] ed [è tradotto in diverse lingue][7], incluso l'italiano!
```
Comment (Madis0) [inline Description/it_it.md:21]: [Mod Menu Helper has not been translated](https://fabulously-optimized.gitbook.io/modpack/readme/language-support#translating-mod-menu-helper) to Italian yet, so the sentence ending is incorrect :P

Comment (glaav) [reply Description/it_it.md:21]: oh right

**Comment by Madis0** [comment]
Thanks!

**Pull request merged**